UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -956,6 +956,26 @@ public string GetExamineText(PlayerScript script = null)
 					$"<color=#b495bf>\n{theyPronoun} has a blank, absent-minded stare and appears completely unresponsive to anything. {theyPronoun} may snap out of it soon.</color>");
 			}
 
+			foreach (BodyPart part in BodyPartList)
+			{
+				if(part.IsSurface){continue;}
+
+				if (part.IsBleeding)
+				{
+					healthString.Append($"<color=red>\n their {part.BodyPartReadableName} is bleeding!</color>");
```

Comment:
All the previous appends have complete sentences, so you'll want to start this with a capital letter. Also use `theyPronoun`, which is already defined for you just above.

Reply:
Done.